app/lib/features/invite_members/pages/invite_page.dart

```diff
@@ -0,0 +1,194 @@
+import 'package:acter/common/providers/room_providers.dart';
+import 'package:acter/common/themes/app_theme.dart';
+import 'package:acter/common/toolkit/buttons/primary_action_button.dart';
+import 'package:acter/common/toolkit/menu_item_widget.dart';
+import 'package:acter/common/utils/routes.dart';
+import 'package:acter_avatar/acter_avatar.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
+
+class InvitePage extends ConsumerWidget {
+  final String roomId;
+
+  const InvitePage({super.key, required this.roomId});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Scaffold(
+      appBar: _buildAppBar(context),
+      body: _buildBody(context, ref),
+    );
+  }
+
+  AppBar _buildAppBar(BuildContext context) {
+    return AppBar(
+      actions: [
+        _buildPendingActionButton(context),
+        const SizedBox(width: 20),
+      ],
+    );
+  }
+
+  Widget _buildPendingActionButton(BuildContext context) {
+    return OutlinedButton(
+      onPressed: () {
+        context.pushNamed(
+          Routes.invitePending.name,
+          queryParameters: {'roomId': roomId.toString()},
+        );
+      },
+      style: OutlinedButton.styleFrom(
+        padding: const EdgeInsets.all(8.0),
+      ),
+      child: Row(
+        children: [
+          const Icon(
+            Icons.person_outline_outlined,
+            size: 18,
+          ),
+          const SizedBox(width: 5),
+          Text(
+            L10n.of(context).pending,
+            style: Theme.of(context).textTheme.bodySmall,
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildBody(BuildContext context, WidgetRef ref) {
+    return SingleChildScrollView(
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: [
+          const SizedBox(height: 10),
+          _buildInviteHeader(context, ref),
+          const SizedBox(height: 20),
+          _buildInviteMethods(context),
+          const SizedBox(height: 20),
+          divider(context),
+          const SizedBox(height: 30),
+          _buildInviteFromCode(context),
+        ],
+      ),
+    );
+  }
+
+  Widget divider(BuildContext context) {
```

Comment:
Can this be extracted to re-usable ? It'd be nice as we don't wanna run into separate dividers widgets being living across different parts...

Reply:
Remove custom divider widget itself and use default one as below.
`const Divider(indent: 70, endIndent: 70),`